jmxtrans-output/jmxtrans-output-logback/src/main/java/com/googlecode/jmxtrans/model/output/KeyOutWriter.java

```diff
@@ -80,14 +78,15 @@
 	protected static final String MAX_LOG_FILE_SIZE = "10MB";
 	protected static final String DEFAULT_LOG_PATTERN = "%msg%n";
 	protected static final String DEFAULT_DELIMITER = "\t";
-	protected Logger logger;
+	protected LogWriter logwriter;
```

Comment:
Why is this `protected` and not `private final` ? Any good reason?

Reply:
I can switch to ```private```, but not ```final``` because it is initialized in a method and not in the constructor. If I do this the compilation fails.

Reply:
Right, I missed that. We should move initialization to the ctor... that validateSetup method is a legacy left over that should disappear at some point. That being said, this is good enough to be merged. I'll merge tonight unless you want to do this additional cleanup.

Reply:
I don't think I'll commit this evening. You can merge.

Thanks for all the comments! This is my first pull request on a open source project and i'm grateful to you for the reviews!